pkg/apis/servicecatalog/types.go

```diff
@@ -149,6 +149,11 @@ type ServiceBrokerStatus struct {
 	OperationStartTime *metav1.Time
 }
 
+// ClusterServiceBrokerStatus represents the current status of a ClusterServiceBroker
+type ClusterServiceBrokerStatus struct {
+	ServiceBrokerStatus
```

Comment:
Why do we need an extra structure instead of renaming `ServiceBrokerStatus`? Is it for the future sharing between cluster- and namespace-scoped types?